components/expansion-panel/src/vwc-expansion-panel.scss

```diff
@@ -50,6 +50,31 @@
 			#{icon-variables.$icon-size-variable-name}: var(#{expansion-panel-variables.$expansion-panel-icon-size-variable-name}, 20px);
 		}
 	}
+	.heading-text {
+		/* stylelint-disable property-no-vendor-prefix */
+		/* stylelint-disable value-no-vendor-prefix */
+		display: -webkit-box;
+		overflow: hidden;
+		flex-grow: 1;
+		-webkit-box-orient: vertical;
+		-webkit-line-clamp: 2;
+		/* stylelint-enable property-no-vendor-prefix */
+		/* stylelint-enable value-no-vendor-prefix */
+		margin-inline-end: auto;
+		text-align: left;
+	}
```

Comment:
can you please open a ticket to discuss the number of lines support?

Reply:
removed the line-trim and opened an issue: https://github.com/Vonage/vivid/issues/1246